### DISCUSSION TITLE
IP Error during setup

### DISCUSSION THREAD

**Discussion by rtuite** [Security-Onion-Solutions/securityonion#6427] [Unsupported Versions] [answered]
I'm a netsec student and I'm tinkering with Security Onion in my home lab, and I'm getting an error "The IP being routed by Linux is not the IP address assigned to the management interface (eth0)." 

I'm comfortable with *nix systems, but this error eludes me. It comes right after configuring the management interface and setting an IP on it. I am using HyperV and have 2 interfaces assigned to the VM. The only thing I found with this error was someone who resolved it by taking the VM down to one interface only and bypassing the 2 NIC requirement. I want to use this system as intended, so I'd like to avoid that if possible. 

Screenshot:
![image](https://user-images.githubusercontent.com/3290219/144332057-d7c1290f-6476-48f2-a948-bbe89484f676.png)

**Comment by rskntroot** [comment, marked as answer]
What system is reporting this error?

My first stop would would validate the network settings using `nmtui` and `ip route`

**Reply by rtuite** [reply]
This is the initial setup, right after giving it a static IP, gateway, and DNS server

**Reply by defensivedepth** [reply]
Make sure that the other interface doesnt have an IP.

**Reply by rtuite** [reply]
> Make sure that the other interface doesn't have an IP.

I did so, I even went so far as to disable the 2nd interface in `nmtui`, but the setup kept failing until I removed the 2nd NIC from the VM entirely and restarted the network configuration. 

I'm curious how that interface caused issues despite the system strongly recommending having it. This was literally 2 or 3 steps into setup on the first startup of the VM after installation from the provided and validated ISO. I had not been given an opportunity to configure the 2nd interface at all. So, if this interface was active, it was not by my hand. I'm assuming there's a way to add and set up the monitoring interface after the fact.

**Comment by dougburks** [comment]
Are you using our Security Onion ISO image or installing CentOS/Ubuntu and then performing a network installation?

If network installation, did you see Step 9 at https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos?

> NOTE: If any interfaces intended to be used for monitoring were automatically configured via DHCP during Ubuntu installation, setup will ask you to remove them from other network management tools. The following steps will be required to ensure the devices are managed by nmcli:
> Remove monitor interface declarations from /etc/netplan/00-installer-config.yaml:
```
sudo netplan apply
sudo touch /etc/NetworkManager/conf.d/10-globally-managed-devices.conf
sudo service network-manager restart
```
> Re-run setup.

**Reply by rtuite** [reply]
This is using the Security Onion ISO

**Reply by dougburks** [reply]
Which version of our ISO image did you download?

When you install our ISO image, it should start out with no network configuration and thus no routing. 

Is it possible that you manually configured networking before running Setup?  

Or is it possible that you ran through Setup, assigned an IP address to one interface, then ran Setup again and assigned an IP address to the other interface?

Are you able to duplicate this issue on a totally fresh installation of the latest ISO image?
https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md
